### PR TITLE
mvp of editor buttons

### PR DESCRIPTION
**Pull request by alex-418**
Adds two buttons to the editor

### CODE REVIEW

File: inc/shortcodes/glossary/class-glossary.php [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+			'no_texturize_shortcodes',
+			function ( $excluded_shortcodes ) {
+				$excluded_shortcodes[] = 'pb_glossary';
+
+				return $excluded_shortcodes;
+			}
+		);
+		// do_shortcode() is registered as a default filter on 'the_content' with a priority of 11.
+		// We need to run $this->glossaryContent() after this, and after attributions which is set to 12, set to 13
+		add_filter( 'the_content', [ $obj, 'glossaryContent' ], 13 );
```
Comment (bdolor) [inline inc/shortcodes/glossary/class-glossary.php]: this function would seek to append glossary content to the end of the content area. This is not the behaviour we are looking for and can be removed.

File: inc/shortcodes/glossary/class-glossary.php [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+				'id' => $term->ID,
+				'content' => $term->post_content,
+			];
+		}
+
+		self::$glossary_terms = $glossary_terms;
+	}
+
+	/**
+	 * If short-code argument is present [pb_glossary id='33']
```
Comment (bdolor) [inline inc/shortcodes/glossary/class-glossary.php]: I don't think we would want to ask that users identify the id of the glossary term. We're asking them only to highlight a word(s).

File: inc/shortcodes/glossary/class-glossary.php [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+	function shortcodeHandler( $atts, $content = '' ) {
+		$retval = '';
+
+		$a = shortcode_atts(
+			[
+				'id' => '',
+			], $atts
+		);
+
+		if ( ! empty( $a ) ) {
```
Comment (bdolor) [inline inc/shortcodes/glossary/class-glossary.php]: the logic here will never resolve to empty if the variable is set immediately before hand.

File: inc/shortcodes/glossary/class-glossary.php [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+		} else {
+			$retval = $this->glossaryContent();
+		}
+
+		return $retval;
+	}
+
+
+	/**
+	 * Returns the HTML for a term if the ID is provided
```
Comment (bdolor) [inline inc/shortcodes/glossary/class-glossary.php]: invalid to look for ID if we're only passing strings

**Review by bdolor — CHANGES_REQUESTED** [review]

File: inc/shortcodes/glossary/class-glossary.php (hunk 2 above)
Comment (alex-418) [reply inc/shortcodes/glossary/class-glossary.php]: This post id will be automagically inserted after they click on the GL button (if the term exists)

File: inc/shortcodes/glossary/class-glossary.php (hunk 3 above)
Comment (alex-418) [reply inc/shortcodes/glossary/class-glossary.php]: fixed by updating this to `if ( ! empty( $a['id'] ) )`

**Review by bdolor — APPROVED** [review]

**Pull request merged**